packages/platform-ios/src/config/__tests__/native_modules.test.ts

```diff
@@ -97,6 +114,7 @@ describeIfSupportedEnv()('native_modules.rb', () => {
   beforeEach(() => {
     runConfig = {
       dependencyConfig: {
+        reactNativePath: '/root/app/node_modules/react_native',
```

Comment:
you can reuse REACT_NATIVE_ROOT and rename `react_native` to `react-native` there

Reply:
Oh, that was my plan! IDK why I didn't replace it here